static/app/components/onboarding/productSelection.tsx

```diff
@@ -0,0 +1,162 @@
+import {Fragment, useCallback, useMemo} from 'react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import Checkbox from 'sentry/components/checkbox';
+import ExternalLink from 'sentry/components/links/externalLink';
+import QuestionTooltip from 'sentry/components/questionTooltip';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useRouter from 'sentry/utils/useRouter';
+import TextBlock from 'sentry/views/settings/components/text/textBlock';
+
+export enum PRODUCT {
+  ERROR_MONITORING = 'error-monitoring',
+  PERFORMANCE_MONITORING = 'performance-monitoring',
+  SESSION_REPLAY = 'session-replay',
+}
+
+export function ProductSelection() {
+  const router = useRouter();
+  const products = useMemo(() => {
+    return Array.isArray(router.location.query.product)
+      ? router.location.query.product
+      : [router.location.query.product];
+  }, [router.location.query.product]);
```

Comment:
Would this helper function be suitable here?
https://github.com/getsentry/sentry/blob/dab313eef7e1f38feac67d02c1dd88f4b2c4535e/static/app/utils/queryString.tsx#L98-L103

Reply:
oh yes...thanks for the suggestion! will update this in a follow-up 🙏